templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -0,0 +1,19 @@
+name: ovs-configuration.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Configures OVS with proper host networking configuration
+  # This service is used to move a physical NIC into OVS and reconfigure OVS to use the host IP
+  Requires=NetworkManager-wait-online.service openvswitch.service
+  After=NetworkManager-wait-online.service openvswitch.service node-valid-hostname.service
+  Before=network-online.target kubelet.service crio.service
+
+  [Service]
+  # Need oneshot to delay kubelet
+  Type=oneshot
```

Comment:
Oneshot does not yet (in the slightly older systemd shipped in rhcos) support retry on failure. Should we put the logic here in the execstart?

Reply:
i would rather build the script to be more robust rather than add logic here